parsons/local_server.py

```diff
@@ -97,7 +97,8 @@ def get_problems():
         with open(PARSONS_CORRECTNESS, "r", encoding="utf8") as f:
             probs_correct = json.loads(f.read())
     except FileNotFoundError:
-        probs_correct = {pname : False for pname in cache[NAMES_TO_PATHS]}
+        probs_correct = {pname : False for pname in cache[PROBLEM_NAMES]}
+        print(f"probs_correct is {probs_correct}")
```

Comment:
Is this print statement used for debugging in case of a file not found? (Making sure its intentionally in here)